build/push-docker-deploy.sh

```diff
@@ -10,12 +10,9 @@ run/local-cluster.sh start
 run/local-cluster.sh stop
 
 docker tag cockroachdb/cockroach:latest cockroachdb/cockroach:${VERSION}
-docker tag cockroachdb/cockroach-dev:latest cockroachdb/cockroach-dev:${VERSION}
 
-for type in {,-dev}; do
-  for version in {latest, ${VERSION}}; do
-    # Pushing to the registry just fails sometimes, so for the time
-    # being just make this a best-effort action.
-    docker push cockroachdb/cockroach${type}:${version} || true
-  done
+for version in {latest, ${VERSION}}; do
+  # Pushing to the registry just fails sometimes, so for the time
+  # being just make this a best-effort action.
+  docker push cockroachdb/cockroach:${version} || true
```

Comment:
Perhaps we should look into nightly builds: https://circleci.com/docs/nightly-builds